bazel/repository_locations.bzl

```diff
@@ -817,11 +817,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "WAVM",
         project_desc = "WebAssembly Virtual Machine",
         project_url = "https://wavm.github.io",
-        version = "c8997ebf154f3b42e688e670a7d0fa045b7a32a0",
-        sha256 = "bf2b2aec8a4c6a5413081c0527cb40dd16cb67e9c74a91f8a82fe1cf27a3c5d5",
-        strip_prefix = "WAVM-{version}",
-        urls = ["https://github.com/WAVM/WAVM/archive/{version}.tar.gz"],
-        release_date = "2021-12-15",
+        version = "2022-05-14",
+        sha256 = "7cfa3d7334c96f89553bb44eeee736a192826a78b4db114042d38d6882748f5b",
+        strip_prefix = "WAVM-nightly-{version}",
+        urls = ["https://github.com/WAVM/WAVM/archive/refs/tags/nightly/{version}.tar.gz"],
```

Comment:
This line can omit `refs/tags/` and should be `https://github.com/WAVM/WAVM/archive/nightly/{version}.tar.gz`

Reply:
Thanks, done. I'll push it once #22469 is merged, to avoid unnecessary CI run.